index.js

```diff
@@ -170,23 +173,36 @@
   var b = {name: 'b'};
   var c = {name: 'c'};
 
+  //  _type :: a -> String
+  var _type = function(x) {
+    return x != null && R.type(x['@@type']) === 'String' ? x['@@type']
+                                                         : R.type(x);
+  };
+
   var _is = function(type, x) {
```

Comment:
Do we want a type signature here? --

    //  _type :: (TypeRep a, b) -> Boolean